FBSimulatorControl/Management/FBSimulatorControl.m

```diff
@@ -115,4 +115,16 @@ - (BOOL)firstRunPreconditionsWithError:(NSError **)error
   return YES;
 }
 
++ (FBSimulatorPool *)poolForConfiguration:(FBSimulatorControlConfiguration *)configuration
+{
+  return [FBSimulatorPool poolWithConfiguration:configuration deviceSet:[self deviceSetForConfiguration:configuration]];
```

Comment:
I want to change this in another PR that changes the startup path so that preconditions can be met ahead of time. `FBSimulatorControl` creates the `DeviceSet` since it is fail-able if for instance the directory isn't writable. A Pool can only be constructed if a `SimDeviceSet` can.